src/LGraphCanvas.ts

```diff
@@ -286,6 +286,9 @@ export class LGraphCanvas {
   }
   // #endregion Legacy accessors
 
+  /**
+   * @deprecated Use {@link LGraphNode.titleFontStyle} instead.
+   */
```

Comment:
There are a few references of `title_text_font` right now so we cannot directly remove this getter.